src/functions/questionnaire/create-quiz/handler.ts

```diff
@@ -0,0 +1,66 @@
+import QuestionnaireService from "src/database/services/quiz-api-service";
+import { middyfy } from "src/libs/lambda";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import type { QuestionnaireModel } from "src/database/schemas/questionnaire/questionnaire";
+import type { ValidatedEventAPIGatewayProxyEvent } from "src/libs/api-gateway";
+
+const dynamoDb = new DocumentClient();
+const questionnaireService = new QuestionnaireService(dynamoDb);
+
+/**
+ * Handler for creating a new questionnaire entry in DynamoDB.
+ *
+ * @param event - API Gateway event containing the request body.
+ * @returns Response object with status code and body.
+ */
+export const createQuizHandler: ValidatedEventAPIGatewayProxyEvent<QuestionnaireModel> = async (event) => {
```

Comment:
It might be that we are using the DB model here when we should be using the schema. For reference see in elastic app repos the src/schema/curation vs the src/database/models/curation. Also note how the schema is used in the createCuration handler as the type for the lambda.

I thin the main reason for this is that it allows us to have some differences between the DB type of entity and the body sent to the lambda describing the entity to be created. E.g. id can be created in the lambda rather than form the UI

Reply:
This should be build. Now its different in our code than it is in elastic

Reply:
Now it its included to this PR. Please check those also.